nixos/modules/virtualisation/docker.nix

```diff
@@ -158,6 +158,15 @@ in
         Docker package to be used in the module.
       '';
     };
+
+    extraPackages = mkOption {
+      type = types.listOf types.package;
+      default = [ ];
+      example = literalExpression "with pkgs; [ crio gnutar gzip ]";
+      description = lib.mdDoc ''
+        Extra packages to add to PATH for the docker daemon process.
```

Comment:
Whats the usecase for gnutar and gzip?

Reply:
criu uses them when making a checkpoint. Looks like it is to save tmpfs: https://github.com/checkpoint-restore/criu/blob/fbe26923e85d7759e44666733a54c5ab951e8aa6/criu/filesystems.c#L420

Reply:
Shouldn't that rather be fixed in criu in addition to this option?

Reply:
Ah yeah good point, that seems more sensible. I made a PR for that ( https://github.com/NixOS/nixpkgs/pull/235588 ) and changed the example in this PR accordingly.